pricing.js

```diff
@@ -61,16 +61,23 @@ function calculateProductPrice(product, employee, selectedOptions) {
 
   switch (product.type) {
     case 'volLife':
-      price = calculateVolLifePrice(product, selectedOptions)
-      employerContribution = getEmployerContribution(product.employerContribution, price)
+      price = this.calculateVolLifePrice(product, selectedOptions)
+      employerContribution = this.getEmployerContribution(product.employerContribution, price)
       return formatPrice(price - employerContribution)
```

Comment:
formatPrice also needs a `this.` as you 1) should be spying on it and 2) should be consistent in your approach